components/layout/Layout.tsx

```diff
@@ -95,6 +129,13 @@ const ContentArea = styled.div`
   @media (${breakUp("lg")}) {
     margin-right: 40px;
   }
+
+  /* Hides ContentArea on smaller screens when SideBar is toggled. Not tabbable when not in view  */
```

Comment:
Content was previously tabbable when sidebar was toggled. This ensures the Content area is hidden when not in view on smaller screens. If the Sidebar is toggled and the screen is expanded, the ContentArea becomes visible even though sidebarToggled state is still active.